phys/module_microphysics_driver.F

```diff
@@ -2764,7 +2779,52 @@ smaw3d, &    !Normalized Galperin instability function for momentum [no units]
          CALL wrf_error_fatal ( wrf_err_message )
 
       END SELECT micro_select
-
+#if (EM_CORE==1)
```

Comment:
@BarryHLynn 
Barry,
Since everyone sees this more public routine, modify the indenting so that you are aligned with what is above (the END SELECT) and what is below (the ENDDO). Have your own nested block indented.